### PR TITLE
restructure docs in opf format

### PR DESCRIPTION
**Pull request by Shreyanand**
## Related Issues and Dependencies

https://github.com/operate-first/operate-first.github.io/issues/275

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

In this PR, we restructure the project docs to match the standard opf format.

### CODE REVIEW

File: docs/contribute.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contribute to the Time Series Project
+
+To get started with familiarizing yourself with the time series project, check how to [Get Started](get-started.md).
+
+1. If you have an notebooks to contribute towards this project, please [open an issue](https://github.com/aicoe-aiops/time-series/issues) on project's [Github Repository](https://github.com/aicoe-aiops/time-series)
```
Comment (aakankshaduggal) [inline docs/contribute.md:5]: nit - an -> any

File: docs/content.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+# Content
+
+## Blog
+
+Check out the [overview](../) page which goes over the motivation of the project.
```
Comment (aakankshaduggal) [inline docs/content.md:6]: I tried this link from your fork, it's giving me an error. Should we just point to the README directly here?

**Review by aakankshaduggal — CHANGES_REQUESTED** [review]
Overall /lgtm
Just minor comments :+1:

**Review by oindrillac — COMMENTED** [review]
/lgtm

File: docs/contribute.md (hunk 1 above)
Comment (Shreyanand) [reply docs/contribute.md]: +1

File: docs/content.md (hunk 2 above)
Comment (Shreyanand) [reply docs/content.md:6]: I had tried pointing it to the README directly but that messes up links on the website. If we use the absolute link, then the links on the README page won't be accessible, so I thought the website should get precedence over the repository...

**Comment by aakankshaduggal** [comment]
Sounds good @Shreyanand /lgtm

**Review by oindrillac — APPROVED** [review]
/lgtm
/approve

**Pull request merged**